net/test/test-libfab.sh.in

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
```

Comment:
What is the difference of this test script and "net/test/test-user.sh.in"?

I see most of them are identical. 
So, can we use the same script?
If needed, we can use some arguments to differentiate them.

Reply:
Initially the arguments of m0nettest are wrong and the end points of lnet and libfab are different.
so created a new test script to run libfab test.

--added an option to chose lnet or libfab in test-user.sh and modified options of lnet as properly.